android/app/src/main/java/covidsafepaths/bt/exposurenotifications/network/DiagnosisKeyUploader.java

```diff
@@ -189,7 +192,8 @@ public DiagnosisKeyUploader(Context context) {
                         .put("hmackey", ExposureKeyModule.hmacKey)
                         .put("padding", randomBase64Data(Util.getRandomNumber()))
                         .put("regions", regionJson)
-                        .put("appPackageName", BuildConfig.ANDROID_APPLICATION_ID);
+                        .put("appPackageName", BuildConfig.ANDROID_APPLICATION_ID)
+                        .put("revisionToken", prefs.getRevisionToken(null));
```

Comment:
Will the null value be turned into the empty string when converted to JSON?

Reply:
It won't be sent if the value is `null`.
Should we send it as an empty string?

Reply:
They do `len(data.RevisionToken)` on the go side, my newly acquired go knowledge is pretty close to 0 but seems to me that they are measuring the len, if I run code with nil it throws an error, but the type of the request is a string so I don't know how the encoding works so I would vote for an empty string.

Reply:
Oh I saw that you added it in the card description. My bad, I didn't read it.
I'll add it now, thanks!